nautobot/docs/release-notes/version-1.3.md

```diff
@@ -1,4 +1,5 @@
 <!-- markdownlint-disable MD024 -->
+<!-- markdownlint-disable MD051 -->
```

Comment:
`INVOKE_NAUTOBOT_LOCAL invoke markdownlint` complains about this, however docker version doesn't, why?